devtools/deployments/opencloud_full/opencloud.yml

```
@@ -58,6 +58,11 @@ services:
       COMPANION_DOMAIN: ${COMPANION_DOMAIN:-companion.opencloud.test}
       # enable to allow using the banned passwords list
       OC_PASSWORD_POLICY_BANNED_PASSWORDS_LIST: banned-password-list.txt
+      IDM_REVASVC_PASSWORD: "admin"
+      AUTH_BASIC_LDAP_BIND_PASSWORD: "admin"
+      USERS_LDAP_BIND_PASSWORD: "admin"
+      GROUPS_LDAP_BIND_PASSWORD: "admin"
+      IDM_LDAPS_ADDR: 0.0.0.0:9235
```

I configured everything on `.compose.test` to prevent collisions with my `.opencloud.test` deployment. The stalwart URL then needs to be set for the groupware:
```suggestion
      IDM_LDAPS_ADDR: 0.0.0.0:9235
      GROUPWARE_JMAP_BASE_URL: https://${STALWART_DOMAIN:-stalwart.opencloud.test}
```